### ISSUE TITLE
Error: Cannot find module '../dist/decaffeinate.cjs'

### ISSUE THREAD

**Issue by dboskovic** [decaffeinate/decaffeinate#264]
When installing this, for some reason bin/decaffeinate references a file that doesn't exist. There's a `../dist/decaffeinate.js` and a `../dist/decaffeinate.mjs` but no .cjs file. I renamed the referenced file in `bin/decaffeinate` and it works fine.

```
module.js:442
    throw err;
    ^

Error: Cannot find module '../dist/decaffeinate.cjs'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/decaffeinate/bin/decaffeinate:3:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)```

**Comment by eventualbuddha** [comment]
Good catch. Release coming soon.

**Comment by eventualbuddha** [comment]
Fixed in v2.13.4.
